### ISSUE TITLE
minikube logs output takes a long long time

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#11266]
even when there is no minikube running 
```
$ minikube profile list

🤹  Exiting due to MK_USAGE_NO_PROFILE: No minikube profile was found. 
💡  Suggestion: 

    You can create one using 'minikube start'.
    

```

on my mac it is taking more than a few minutes and sitll running
```
real	5m31.022s
user	0m4.015s
sys	0m5.576s
```

**Comment by spowelljr** [comment]
I'm a little confused by this, do you mean `logs` or `profile`? In your example you're running `profile` but the title says `logs`.

I ran `minikube profile list` and it was very fast.
```
% time ./out/minikube profile list

🤹  Exiting due to MK_USAGE_NO_PROFILE: No minikube profile was found. 
💡  Suggestion: 

    You can create one using 'minikube start'.
    

./out/minikube profile list  0.20s user 0.10s system 106% cpu 0.279 total
```

**Comment by andriyDev** [comment]
Seems related to #10854.

**Comment by medyagh** [comment]
seems like @spowelljr has a fix for this, will fix for 1.20.0
